snippets/visualbasic/VS_Snippets_CLR/InstallContext_InstallContext/VB/installcontext_installcontext.vb

```diff
@@ -79,7 +79,7 @@ Namespace MyInstallContextNamespace
          If args.Length < 2 Then
 ' <Snippet2>
             ' There are no command line arguments, create an empty 'InstallContext'.
-            myInstallObject.myInstallContext = New InstallContext()
+            myInstallObject.Context = New InstallContext()
```

Comment:
I'm not sure this is correct. InstallContext_Example defines a field called myInstallContext on line 29.

Reply:
Right, there is no property or field named `Context`